### PR TITLE
Fix hook-me script

### PR DESCRIPTION
**Pull request by kon-angelo**
**How to categorize this PR?**

/area dev-productivity
/kind regression
/platform aws

**What this PR does / why we need it**:

Adapt scripts after last g/g vendoring

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```other operator
NONE
```

### CODE REVIEW

**Review by timuthy — APPROVED** [review]
Thank you 🙂 
/lgtm

File: Makefile [hunk 1]
```diff
@@ -68,18 +68,28 @@ start:
...
 		-ldflags $(LD_FLAGS) \
 		./cmd/$(EXTENSION_PREFIX)-$(ADMISSION_NAME) \
 		--webhook-config-server-host=0.0.0.0 \
 		--webhook-config-server-port=$(WEBHOOK_CONFIG_PORT) \
 		--webhook-config-mode=$(WEBHOOK_CONFIG_MODE) \
         $(WEBHOOK_PARAM)
 
+.PHONY: hook-me
+hook-me:
+	bash $(GARDENER_HACK_DIR)/hook-me.sh $(EXTENSION_NAMESPACE) $$(kubectl get namespace | grep $(NAME) | head -n1| cut -d" " -f1) $(WEBHOOK_CONFIG_PORT)
```
Comment (timuthy) [inline Makefile:91]: No it's inconsistent to https://github.com/gardener/gardener-extension-provider-openstack/blob/4531784c98d2e6339bc94f296f56484bb83bfa37/Makefile#L90. Why do we do it differently here?

Comment (kon-angelo) [reply Makefile:91]: This has a bit more smart to find the proper namespace of the extension in a seed which usually has a suffix. It is not particularly smart but it does a better job at targeting the correct ns.

Comment (timuthy) [inline Makefile:91]:
```suggestion
hook-me: $(KUBECTL)
	bash $(GARDENER_HACK_DIR)/hook-me.sh $(EXTENSION_NAMESPACE) $$(kubectl get namespace -o custom-columns=NAME:.metadata.name | grep $(NAME) | head -n1) $(WEBHOOK_CONFIG_PORT)
```

**Review by timuthy — CHANGES_REQUESTED** [review]

**Pull request merged**